test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties_UI_SetGlobalProperties_SUCCESS

```

Comment:
@vvvakulenko, in the requirement is listed RPC: PerformAudioPathThru, but in MOBILE/HMI_API.xml such RPC is not specified.
Please clarify is this RPC PerformAudioPassThru or it should be some new RPC.